src/components/inputs/z-input-message/index.tsx

```diff
@@ -21,12 +21,22 @@ export class ZInputMessage {
     warning: "exclamation-circle",
   };
 
+  private statusRole = {};
+
+  @Watch("message")
+  onMessageChange(): void {
+    this.statusRole = {role: "alert"};
```

Comment:
Secondo me qui bisogna controllare che il messaggio ci sia.
Di solito questo componente è sempre renderizzato sotto lo ZInput, e ogni tanto viene riempito col messaggio.. Quindi secondo me il role alert bisogna aggiungerlo quando compare il messaggio e rimuoverlo quando scompare.

Reply:
Il fatto è che uno dei casi limite segnalati da LIA è proprio quello in cui abbiamo un `z-input-message` al caricamento pagina con messaggio presente. Io mi aspetto che in qualsiasi caso il `role="alert"` si attivi nel momento in cui un messaggio cambia, richiamando l'attenzione dell'utente

Reply:
Infatti io ho detto di togliere l'alert quando il messaggio non c'è.
Se il messaggio è "messaggio" e poi diventa "", passo comunque da lì e setto il role alert, ma in quel caso non ci deve essere. Non so se mi sono spiegata.